lib/state/state.go

```diff
@@ -144,12 +144,40 @@ func LogDir(baseDir string, suffixes ...string) string {
 	return filepath.Join(append(elems, suffixes...)...)
 }
 
+// GravityInstallerServicePath returns the path to the installer service
+// as configured in the state directory rooted at baseDir.
+func GravityInstallerServicePath(baseDir string) (path string, err error) {
```

Comment:
Is this function actually used anywhere? Couldn't find it.

Reply:
Yes, it is used [here](https://github.com/gravitational/gravity/blob/dmitri/7.0.x/2009-service-state/tool/gravity/cli/install.go#L954)